<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateReaderDialog</class>
 <widget class="QDialog" name="UpdateReaderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>读者ID：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>读者姓名：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="readerNameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>读者性别：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>电话号码：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="telLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>欠款：</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDoubleSpinBox" name="keepmoneyDoubleSpinBox">
     <property name="maximum">
      <double>9999999999.989999771118164</double>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>ok</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="sexComboBox">
     <item>
      <property name="text">
       <string>男</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>女</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>其他</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>IDLineEdit</tabstop>
  <tabstop>readerNameLineEdit</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>telLineEdit</tabstop>
  <tabstop>keepmoneyDoubleSpinBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateReaderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>UpdateReaderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>166</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
